nixos/modules/virtualisation/cri-o.nix

```diff
@@ -126,7 +126,7 @@ in
       documentation = [ "https://github.com/cri-o/cri-o" ];
       wantedBy = [ "multi-user.target" ];
       after = [ "network.target" ];
-      path = [ cfg.package ];
+      path = [ cfg.package pkgs.conntrack-tools ] ++ optional (cfg.storageDriver == "zfs") pkgs.zfs;
```

Comment:
`conntrack-tools` can be added to the [`cri-o`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/virtualization/cri-o/wrapper.nix) wrapper.

I guess `zfs` is fine here as it's optional unless it can be passed to `extraPackages` from the module?

Reply:
I don't think that this will work with `zfsUnstable`/`enableUnstable`.

There should be an internal option in the zfs module indicating the package used.
That package should also be used here.